src/server/models/mediaquery.js

```diff
@@ -176,3 +177,49 @@ module.exports.setCrosslists = async (crosslists, collectionName) => {
     throw err;
   }
 };
+
+module.exports.updatePlayback = async (obj, collectionName) => {
+  const playbackCollection = client.db(DB_DATABASE).collection(collectionName);
+  const session = client.client.startSession();
+  session.startTransaction();
+  try {
+    const { userid, tab, file, classShortname } = obj;
+    const query = {
+      userid,
+      tab,
+      file,
+      classShortname,
+    };
+    const deleteResult = await playbackCollection.deleteMany(query, {
```

Comment:
@maxpluez Should try to do this in one database operation. MongoDB has an upsert feature that if the record is found it will update, else it will insert: https://docs.mongodb.com/manual/reference/method/Bulk.find.upsert/